### DISCUSSION TITLE
How to change the number of output images?

### DISCUSSION THREAD

**Discussion by SuperGamer001** [lllyasviel/Fooocus#3744] [Q&A] [answered]
I want to allow my system to make more images then just two. Is there a way where I can add more to the number of outputs? I believe I'm using version 2.5.5 Also I'm new to GitHub, so if I need some correcting, let me know. 👍🏻

**Comment by ghost** [comment, marked as answer]
Just click "Advanced".

**Reply by SuperGamer001** [reply]
I missed that completely! I literally overlooked that.

**Comment by Yogesh-DevHub** [comment]
In advance section you can select the maximum number of outputs which is 32. 
If you want to increase higher then 32, change in the config file: https://youtu.be/3I-Rl6sCLiA?si=CsZXjY-uSmitL4zI

**Reply by SuperGamer001** [reply]
I just saw this when I marked the answer, but yes. This is the answer I was looking for. I think 32 is more then enough, but I'll look into this if I do want more. 👍🏻👍🏻
